src/transformers/models/layoutlmv2/processing_layoutlmv2.py

```diff
@@ -71,23 +102,9 @@ def __call__(
         images,
         text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
         text_pair: Optional[Union[PreTokenizedInput, List[PreTokenizedInput]]] = None,
-        boxes: Union[List[List[int]], List[List[List[int]]]] = None,
+        boxes: Optional[Union[List[List[int]], List[List[List[int]]]]] = None,
```

Comment:
We can put this in ImageKwargs as the general consensus was to leave only text, image, video and audio as args in processors. Same for `text_pair` and `word_labels` (see https://github.com/huggingface/transformers/pull/32544#discussion_r1714059235)

Reply:
@zucchini-nlp there were 3 args (`text_pair`, `boxes`, & `word_labels`) I needed to replace but only had `audio` & `videos` so I decided to also add `backwards_compatibility_placeholder_arg` & raised warnings for usage.

Reply:
resolving as this has now been removed.

I've also implemented a new approach which I've described in more detail here: https://github.com/huggingface/transformers/issues/31911#issuecomment-2294820439